.github/workflows/slash-beekeeper.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           command: beekeeper
           reaction-type: "rocket"
       - name: Get PR Branch
-        if: steps.command.outputs.command-arguments == 'beekeeper'
+        if: steps.command.outputs.command-name == 'beekeeper'
```

Comment:
is this needed?

Reply:
Not actually, as it was merged to master recently.